components/FMenu.vue

```diff
@@ -107,7 +107,8 @@
   border-radius rem(16)
   elevation(2)
   color var(--fmenu--text-color)
-  padding rem(8)
+  padding rem(16)
+  font-size rem(14)
```

Comment:
I took the liberty to change the `font-size` here because we already do the same thing (without using use-font) for a `.FMenu__option` :

<pre>
.FMenu__option
  display flex
  align-items center
  flex-wrap wrap
  border-radius rem(16)
  min-height rem(24)
  padding rem(12) rem(8)
  color var(--fmenu--text-color)
  <b>font-size rem(14)</b>
  transition background 300ms, color 300ms
  cursor pointer
</pre>

That's why I thought it would be more consistent within the component, tell me if you agree !

Reply:
I aggree ;-)